core/backend/src/perftest/TilesGenUtils.ts

```diff
@@ -63,11 +63,12 @@ export class BackendTileGenerator {
     maximumMajorTileFormatVersion: CurrentImdlVersion.Major,
     enableInstancing: true,
     enableImprovedElision: true,
-    useProjectExtents: false,
+    useProjectExtents: true,
```

Comment:
Just double-checking this is intentional.

Reply:
Yes - I want it on by default on master (3.0). I expect to backport to 2.19.x and turn it off by default there.